app/routes/manifest[.]webmanifest.ts

```diff
@@ -0,0 +1,75 @@
+import { json } from "@remix-run/node";
+
+export const loader = async () =>
+  json(
+    {
+      short_name: "Shelf",
+      name: "Shelf | Asset Management Made Simple",
+      start_url: "/",
+      display: "standalone",
+      background_color: "#d3d7dd",
```

Comment:
These are not our colors. Please adjust it to the correct colors. You can ask @jurrejansen if you need input about this.